src/app/markers.ts

```diff
@@ -85,6 +99,7 @@ export function canVisitFlower(
   recentFlowerInteractions: Interaction[],
 ): boolean {
   const haveVisitedThisFlower = recentFlowerInteractions
+    .filter(interaction => !interaction.incompatibleFlower)
```

Comment:
Oh this is interesting, so unlike normal flowers, incompatible flowers aren't affected by the "pollen cycle" lockout.

Reply:
Yeah. I was wondering why the last interaction logic wasn't working, but the interaction was disabled simply due to incompatible Flower still being a recent interaction.